not_my_ex/settings.py

```diff
@@ -15,6 +15,8 @@ class EnvironmentVariableNotFoundError(Exception):
 MASTODON_INSTANCE = getenv("NOT_MY_EX_MASTODON_INSTANCE", "https://mastodon.social")
 MASTODON_TOKEN = getenv("NOT_MY_EX_MASTODON_TOKEN")
 
+DEFAULT_LANG = getenv("NOT_MY_EX_DEFAULT_LANG", None)
```

Comment:
`None` is already [`getenv's default](https://docs.python.org/3/library/os.html#os.getenv), so maybe we can write less code ; )

```suggestion
DEFAULT_LANG = getenv("NOT_MY_EX_DEFAULT_LANG")
```